build.gradle

```diff
@@ -139,7 +139,7 @@ gradle.projectsEvaluated {
     }
 
     def mavenProjectDescription = {
-      name project.mavenName ?: project.name
+      name = project.mavenName ?: project.name
```

Comment:
> 'name' cannot be applied to '(java.lang.Object)' : 142, 151, 161, 171, 176, 182

These warnings also seem to be wrong. But I won't object.

Reply:
I assume it is like in https://stackoverflow.com/questions/32905270/intellij-idea-and-gradle-cannot-be-applied-to-groovy-lang-closure which pints to http://groovy-lang.org/metaprogramming.html#_methodmissing .